postcss.config.js

```diff
@@ -17,6 +17,6 @@ module.exports = {
         'focus-visible-pseudo-class': false
       }
     }),
-    require('cssnano'),
+    process.env.CI === 'true' ? require('cssnano') : null
```

Comment:
I'm trying to get the CSS to not be compressed output for the new panel, my thinking here is to set to `null` in the proper circumstances.

Reply:
What if we just read in the un-compiled .pcss file instead of the compiled .css file?

Reply:
Was thinking if we're sharing the compiled CSS it would be easier to copy/paste to a prototyping tool like Codepen.